web-api/src/Kerbero.WebApi/Controllers/SmartLockKeysController.cs

```diff
@@ -0,0 +1,69 @@
+using Kerbero.Domain.Common.Models;
+using Kerbero.Domain.SmartLockKeys.Dtos;
+using Kerbero.Domain.SmartLockKeys.Interfaces;
+using Kerbero.WebApi.Extensions;
+using Kerbero.WebApi.Models.Requests;
+using Kerbero.WebApi.Utils.Extensions;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Kerbero.WebApi.Controllers;
+
+[Authorize]
+[ApiController]
+[Route("api/[controller]")]
+public class SmartLockKeysController: ControllerBase
+{
+	private readonly ICreateSmartLockKeyInteractor _createSmartLockKeyInteractor;
+	private readonly IGetSmartLockKeysInteractor _getSmartLockKeysInteractor;
+
+	public SmartLockKeysController(
+		ICreateSmartLockKeyInteractor createSmartLockKeyInteractor, 
+		IGetSmartLockKeysInteractor getSmartLockKeysInteractor
+		)
+	{
+		_createSmartLockKeyInteractor = createSmartLockKeyInteractor;
+		_getSmartLockKeysInteractor = getSmartLockKeysInteractor;
+	}
+
+	[HttpPost]
+	public async Task<ActionResult<SmartLockKeyDto>> CreateSmartLockKeyBySmartLockId([FromBody] CreateSmartLockKeyRequest request)
```

Comment:
```suggestion
	public async Task<ActionResult<SmartLockKeyDto>> CreateSmartLockKeyBySmartLockId([FromBody] CreateSmartLockKeyRequest request)
```
Is `[FromBody] ` needed?